native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/email-native/resource-config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "resources":{
+  "includes":[
+    {
+      "pattern":"\\QMETA-INF/javamail.charset.map\\E"
+    }, 
+    {
+      "pattern":"\\QMETA-INF/javamail.default.address.map\\E"
+    }, 
+    {
+      "pattern":"\\QMETA-INF/mailcap.default\\E"
+    }, 
+    {
+      "pattern":"\\QMETA-INF/mailcap\\E"
+    }, 
+    {
+      "pattern":"\\QMETA-INF/services/javax.mail.Provider\\E"
+    },
+    {
+      "pattern":"\\Qorg/slf4j/impl/StaticLoggerBinder.class\\E"
+    }
+  ]},
+  "bundles":[]
```

Comment:
We can remove bundles since this is empty

Reply:
Fixed with [f4700cc4b38824cd5bf4c1c3148247a3dc32a58a](https://github.com/ballerina-platform/module-ballerina-email/pull/561/commits/f4700cc4b38824cd5bf4c1c3148247a3dc32a58a)